Router/Destination.swift

```diff
@@ -31,12 +31,11 @@ extension PresentationHint: CustomStringConvertible {
         switch self {
         case .navigate:
             return "navigate"
-        case .modal(let metrics):
-            let style = metrics.style != nil ? String(describing: metrics.style!) : "none"
-            return "modal - animated: \(metrics.animated), style: \(style)"
-        case .tab(let index):
+        case let .modal(metrics):
+            return "modal - animated: \(metrics.animated), style: \(metrics.style.styleDescription)"
+        case let .tab(index):
             return "tab - index: \(index)"
-        case .replaceTab(let index, let shouldSwitch):
+        case let .replaceTab(index, shouldSwitch):
```

Comment:
Noisy code style changes here...

Reply:
I can change that back if you want.

Reply:
It probably doesn't matter. I should probably start adopting the style anyway.